lib/active_merchant/billing/gateways/ogone.rb

```diff
@@ -285,8 +284,6 @@ def add_d3d(post, options)
           THREE_D_SECURE_DISPLAY_WAYS[options[:win_3ds]] :
           THREE_D_SECURE_DISPLAY_WAYS[:main_window]
         add_pair post, 'WIN3DS', win_3ds
-
-        add_pair post, 'HTTP_ACCEPT',     options[:http_accept] || '*/*'
```

Comment:
❓ Just for curiosity, what is the reason to remove this line ?

Reply:
It was moved to another line, to be used for 3dsv1 cases